### PR TITLE
feat: create post

### PR DESCRIPTION
**Pull request by tushargoyal812**
https://user-images.githubusercontent.com/69681896/168018592-590743b0-b6d3-4dc4-b56c-7feb5de39ec3.mp4

### CODE REVIEW

File: my-tweet/src/pages/Home.js [hunk 1]
```diff
@@ -1,15 +1,19 @@
...
     return(
-        <Wrap my='3rem' mx='12rem'>
+        <>
+        <Navbar/>
+        <Wrap mx='12rem'>
         <Flex>
         <Sidebar/>
         <PostSection/>
-        <Sidebar/>
+        {/* <Sidebar/> */}
```
Comment (deepak29-git) [inline my-tweet/src/pages/Home.js:14]: remove commented code

**Review by deepak29-git — APPROVED** [review]
code organized well and variable name is also good

**Review by sharmayatendra — APPROVED** [review]
code looks clean
design is consistent 
good file structure

File: my-tweet/src/pages/Home.js (hunk 1 above)
Comment (tushargoyal812) [reply my-tweet/src/pages/Home.js:14]: ok

**Pull request merged**